build.sh

```diff
@@ -8,20 +8,24 @@ set -x
 set -o pipefail
 
 # Make sure to start with a clean 'manifests' dir
-rm -rf environments/kubernetes/manifests
-mkdir environments/kubernetes/manifests
+rm -rf environments/dev/manifests
+mkdir environments/dev/manifests
 
-jsonnet -J vendor -m environments/kubernetes/manifests environments/kubernetes/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
-find environments/kubernetes/manifests -type f ! -name '*.yaml' -delete
+jsonnet -J vendor -m environments/dev/manifests environments/dev/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
```

Comment:
Since this is literally the same contents as build_dev.sh, it might be better to do `./build_dev.sh` so we don't have to keep two files in sync